lib/checks/label/label-content-name-mismatch.js

```diff
@@ -0,0 +1,34 @@
+function getAriaText(elm) {
+	if (elm.getAttribute('aria-label')) {
+		return elm
+			.getAttribute('aria-label')
+			.toLowerCase()
+			.trim();
+	}
+	if (elm.getAttribute('aria-labelledby')) {
+		return elm
+			.getAttribute('aria-labelledby')
+			.toLowerCase()
+			.trim();
+	}
+	return null;
+}
+
+const accessibleText = getAriaText(node);
+// if no accessible text - fail
+if (!accessibleText || !accessibleText.length) {
+	return false;
+}
+
+const contentText = node.textContent.toLowerCase().trim();
```

Comment:
You should use `text.sanitize`. It takes things like double spaces into account as well. Related question, does textContent solve for HTML entities like `&nbsp;`? We should probably have a test that makes sure HTML Entities aren't seen as different characters.